### PR TITLE
docs: add yarn and pnpm install commands

### PR DESCRIPTION
**Pull request by import-brain**
This might seem a bit unnecessary, but I think it would be a nice quality-of-life addition for the yarn users out there, they can just copy-paste from the doc.

### CODE REVIEW

**Review by damienwebdev — DISMISSED** [review]
lgtm

**Comment by Shinigami92** [comment]
pnpm also gets really popular and I also have all three package managers in my own project docs!
So also add it, all three or only npm or e.g. <packageManager> (but this is not copy-paste-able 👎)

File: docs/guide/index.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Install it as a Dev Dependency using your favorite package manager.
 ```shell
 npm install @faker-js/faker --save-dev
 ```
+or
+```shell
+yarn add @faker-js/faker --dev
+```
```
Comment (Shinigami92) [inline docs/guide/index.md:30]:
````suggestion

or

```shell
yarn add @faker-js/faker --dev
```

or

```shell
pnpm add @faker-js/faker --save-dev
```
````

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Comment by import-brain** [comment]
@Shinigami92 Wow, until you mentioned it, I have never heard of pnpm.

Also, I can copy-paste the commands in the code blocks to my terminal just fine, what do you mean by "this is not copy-paste-able"?

**Comment by Shinigami92** [comment]
> Also, I can copy-paste the commands in the code blocks to my terminal just fine, what do you mean by "this is not copy-paste-able"?

I just mean when we would write `<packageManager> add ...`

**Comment by import-brain** [comment]
> > Also, I can copy-paste the commands in the code blocks to my terminal just fine, what do you mean by "this is not copy-paste-able"?
> 
> I just mean when we would write `<packageManager> add ...`

Oh, I see what you mean, thanks for the clarification.

**Comment by fncolon** [comment]
I'm not reviewer, but I think you should sync the installation guide in README too.

**Review by Shinigami92 — APPROVED** [review]

**Review by clarkerican — APPROVED** [review]

**Pull request merged**